packages/wallets/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "@deriv/wallets",
+    "private": true,
+    "version": "1.0.0",
+    "main": "src/index.ts",
+    "dependencies": {
+        "@deriv/api": "^1.0.0",
```

Comment:
We gonna try to only depend on `@deriv/api` to avoid using existing legacy code as much as possible.